gui_main.py

```diff
@@ -187,16 +187,30 @@ def __init__(self, top=None):
         self.current_color = StringVar(top)
         self.current_color.set('Red')
         self.colors = {'Red', 'Blue', 'Cyan', 'Black', 'Green'}
-        self.dropdown_menu = OptionMenu(top, self.current_color,
+        self.dropdown_menu_color = OptionMenu(top, self.current_color,
                                         *self.colors, command=self.Dropdown_Changed)
-        self.dropdown_menu.pack(side='top', anchor='w')
-        self.dropdown_menu.place(relx=0.78, rely=0.40, height=18, width=100)
-        self.dropdown_menu.configure(activebackground=_activebgcolordark)
-        self.dropdown_menu.configure(background=_bgcolorlight)
-        self.dropdown_menu.configure(fg=_fgcolorlight)
-        self.dropdown_menu['menu'].configure(activebackground=_activebgcolordark)
-        self.dropdown_menu['menu'].configure(background=_bgcolorlight)
-        self.dropdown_menu['menu'].configure(fg=_fgcolorlight)
+        self.dropdown_menu_color.pack(side='top', anchor='w')
+        self.dropdown_menu_color.place(relx=0.78, rely=0.40, height=18, width=100)
+        self.dropdown_menu_color.configure(activebackground=_activebgcolordark)
+        self.dropdown_menu_color.configure(background=_bgcolorlight)
+        self.dropdown_menu_color.configure(fg=_fgcolorlight)
+        self.dropdown_menu_color['menu'].configure(activebackground=_activebgcolordark)
+        self.dropdown_menu_color['menu'].configure(background=_bgcolorlight)
+        self.dropdown_menu_color['menu'].configure(fg=_fgcolorlight)
+
+        self.current_line_style = StringVar(top)
+        self.current_line_style.set('-')
+        self.line_styles = {'-', '--', '-.', ':'}
```

Comment:
This shows only symbols in the drop down which looks a bit weird. Please show style names on drop down. Like solid line, dashed, dot dashed. Symbols should be present in bracket.